docker/scripts/cli.ts

```diff
@@ -86,6 +86,7 @@ yargs(process.argv.slice(2))
             handleCliCommand({
               success: false,
               error: new Error('Either provide the release Git branch or mount the Airnode source code directory'),
+              data: undefined,
```

Comment:
@Siegrift this (and the same below) is necessary because of your PR 😆  https://github.com/api3dao/promise-utils/pull/90